ngraph/src/ngraph/pass/nop_elimination.cpp

```diff
@@ -254,7 +254,17 @@ static bool eliminate_unsqueeze(const std::shared_ptr<Node>& node)
 
     auto unsqueeze = as_type_ptr<opset3::Unsqueeze>(node);
     auto input = unsqueeze->input_value(0).get_node_shared_ptr();
-    auto data_shape = input->input_value(0).get_partial_shape();
+
+    PartialShape data_shape;
+    if (op::is_parameter(input))
```

Comment:
This is strange because `replace_node_update_name` must not remove node if it has Parameter as input.
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/src/ngraph/graph_util.cpp#L891

Reply:
This change resolve a rare issue when input is a parameter because it has no inputs so `input->input_value(0)` return a nullptr AFAIK.

Reply:
This fix was introduced in #1206

Reply:
Why we need to support this cases when op has no inputs? Inputs required by spec and if they disappeared then Function is not correct in this case and we need to find out where is this happens.

Reply:
@tsocha Yes, the problem was when we try get input of input of Squeeze op if the input of Squeeze is parameter as in model below:
![image](https://user-images.githubusercontent.com/11331953/89289321-55e82f00-d657-11ea-807c-c35b2f56f76e.png)

Reply:
In #1206 was implemented solution to `eliminate_squeeze`, here to `eliminate_unsqueeze`

Reply:
`data_shape` is used only when unsqueeze input is suqeeze, so lets move `data_shape` inside `if (squeeze)` and `if (unsqueeze_i)`  statements where we don't have to check is it parameter or not.

Reply:
Also don't forget to write tests for case you fixed

Reply:
done